airgun/entities/product.py

```diff
@@ -39,13 +38,6 @@ def delete(self, entity_name):
         view.dialog.confirm()
         view.flash.assert_no_error()
         view.flash.dismiss()
-        view = self.navigate_to(self, 'All')
```

Comment:
Why to require to remove this one ? Am i missing something ?

Reply:
we don't want explicitly navigate to 'All' view if we are not going to do anything there?

Reply:
ahh .. make sense ..